measureme/src/raw_event.rs

```diff
@@ -12,40 +12,44 @@ pub struct RawEvent {
     pub event_id: EventId,
     pub thread_id: u32,
 
-    // The following 96 bits store the start and the end timestamp, using
+    // The following 96 bits store the start and the end counter value, using
     // 48 bits for each.
+    // FIXME(eddyb) s/time/count/
     pub start_time_lower: u32,
+    // FIXME(eddyb) s/time/count/
     pub end_time_lower: u32,
     pub start_and_end_upper: u32,
 }
 
-/// `RawEvents` that have an end time stamp with this value are instant events.
-const INSTANT_TIMESTAMP_MARKER: u64 = 0xFFFF_FFFF_FFFF;
+/// `RawEvents` that have an end counter value with this value are instant events.
+const INSTANT_COUNT_MARKER: u64 = 0xFFFF_FFFF_FFFF;
 
-/// The max instant timestamp we can represent with the 48 bits available.
+/// The max instant counter value we can represent with the 48 bits available.
+// FIXME(eddyb) s/TIMESTAMP/COUNT/
```

Comment:
This is a note for me to create issues to clean this up at the next major version.